requests-for-comments/2023-10-27-base3000-launch.md

```diff
@@ -0,0 +1,70 @@
+# Base 3000 & Notebooks launch
+
+We want to launch the ‘first version’ of PostHog 3000 after the next sprint, which means the week starting November 13th (+ 1 week if needed). We have decided to **include Notebooks** in this launch, so that we can have a more impactful launch that combines functionality and design changes.
+
+## Scope for launch
+
+- Dark Mode
+- Base 3000 [Check this mega issue for details](https://github.com/PostHog/posthog/issues/18089)
+- Notebooks side panel
+    
+<img width="674" alt="Screenshot 2023-10-27 at 10 38 50" src="https://github.com/PostHog/meta/assets/14750837/abb9e559-91ea-408b-a8fd-5105f3c4c626">
+
+## Out of scope for launch (amongst other things)
+
+- Sidebar (Index view)
+- Canvas-powered dashboards
+- Query builder
+- [Check this mega issue for details + the full list](https://github.com/PostHog/posthog/issues/18089)
+
+
+## Rollout plan
+- [ ] Base 3000 ready to release (10th Nov)
+- [ ] Internal and a/b testing starts (13th Nov)
+- [ ] Docs revamp and updates ready (24th Nov)
+- [ ] Product pages for 3000 / Notebooks (24th Nov)
+- [ ] Release [PostHog 3000 (Base 3000 scope)](https://app.posthog.com/feature_flags/8472) (27th Nov)
+- [ ] Release [Notebooks](https://app.posthog.com/feature_flags/8734) (4th Dec) (or together with Base 3000 if we feel docs are easier)
+
+
+## Pre-launch evaluation
+
+We decided that even the "base" version has enough major changes that we want to test that it both doesn't impact our key metrics negatively as well as (if possible) evaluate if it is overall an improvement, especially with new users.
+This might be hard to do or to get clear signals for whether it is an improvement (especially as it is only a step along the way to the end goal UI/UX).
+
+- [ ] **Signup/onboarding Experiment**: _just_ for the onboarding we could experiment with the styles separate from the rest of the app
+  - [ ] **Metrics**: Same as usual onboarding metrics?
+  - [ ] Important that we keep the style consistent from signup through to app usage as otherwise it will be a janky experience
+  - [ ] We mostly care about **negatively** impacting the flow
+- [ ] **General Experiment**: in combination with internal testing we could start to roll out the product to new and existing users as an experiment
+  - [ ] Important that we keep the style consistent from signup through to app usage as otherwise it will be a janky experience
+  - [ ] We originally opted against being able to go back to the old UI - do we want to offer this?
+  - [ ] **Metrics**:
+    - [ ] Same as onboarding?
+    - [ ] Retention over time (what time frame do we need to be clear on this?)
+    - [ ] Number of clicks on the sidebar as a negative metric?
+    - [ ] Number of searches now vs before?
+    - [ ] Choice to use dark mode?
+- [ ] **Qualatitive testing**
+  - [ ] Survey for opted in users witht the new UI to ask what they think about it?
+  - [ ] In person feedback
```

Comment:
I am on vacation the second half of November, so I will unlikely be able to lead in-person interviews/usability tests, but I can reach out to users I have interviewed in the past I know would be more than happy to give feedback on the new UI, and can set up the calls for when I am away, so that you can conduct them.

Reply:
We can also invite those users we interviewed / surveyed to a temporary shared Slack channel, where they can continue to share feedback or design bugs, if they see them. This might also be a good opportunity to find *any* design/UI bugs we have and are not aware of, and by running this as a temporary project users might be more inclined to share feedback as well